backend/utils/airtable/utils.js

```diff
@@ -70,6 +70,14 @@ async function getCurrentRoundNumber() {
   return roundParameters ? roundParameters[0].fields["Round"] : -1;
 }
 
+async function getCurrentCategoryId() {
```

Comment:
Maybe getCurrentCategoryId() should be named getCurrentDiscourseCategoryId()? Category is also used in proposals.

Reply:
Done!